cargo-near-build/src/fs.rs

```diff
@@ -4,6 +4,7 @@ use eyre::WrapErr;
 /// Copy a file to a destination.
 ///
 /// Does nothing if the destination is the same as the source to avoid truncating the file.
+#[cfg(feature = "build_internal")]
```

Comment:
How do you feel about feature-flagging the whole module instead of individual functions?